test/support/utils_sycl.h

```diff
@@ -133,12 +133,11 @@ make_new_policy(_Policy&& __policy)
 #    endif // ONEDPL_USE_PREDEFINED_POLICIES
 #endif     // ONEDPL_FPGA_DEVICE
 
-// create the queue with custom asynchronous exceptions handler
-static auto my_queue = sycl::queue(default_selector, async_handler);
-
 inline
-sycl::queue get_test_queue()
+sycl::queue& get_test_queue()
```

Comment:
I think the return type should be changed in `test/support/utils_invoke.h:103` as well (see failing tests).

Reply:
As for me - it's not required because sycl::queue has pimpl-implementation.

Reply:
There used to be a mismatch between the definition and a forward declaration in another header. It must have been fixed to allow the test to compile.

Reply:
@SergeyKopienko, now I got what you suggested. The suggestion was about returning a copy in both declarations. It initially seemed to me as if it was fine to leave a copy at `test/support/utils_invoke.h:103` while having a reference here.